astropy/units/imperial.py

```diff
@@ -2,8 +2,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 """
-This package defines colloquially used Imperial units.  By default, they
-are not enabled.  To enable them, do::
+This package defines colloquially used Imperial units.  They are
```

Comment:
yes, that is intentional, since the `cds` units replace some of the normal ones, so one cannot have them easily side-by-side.